Dockerfile

```diff
@@ -6,103 +6,131 @@ ARG NCPUS=1
 
 # R version 3.5.3 is not available via Conda, sticking to 3.5.1
 # ENV R_VERSION=${R_VERSION:-3.5.1}
-ENV R_VERSION=3.5.1
-ENV PY_VERSION=${PY_VERSION:-3.7}
-
-ENV USER=rstudio
-ENV LC_ALL=en_GB.UTF-8 \
-    LANG=en_GB.UTF-8
-
-ENV PATH /opt/conda/bin:$PATH
+ENV R_VERSION=3.5.1 \
+    PY_VERSION=${PY_VERSION:-3.7} \
+    USER=rstudio \
+    LC_ALL=en_GB.UTF-8 \
+    LANG=en_GB.UTF-8 \
+    PATH=/opt/conda/bin:$PATH
 
 ## Set locale & install packages
 RUN echo "en_GB.UTF-8 UTF-8" >> /etc/locale.gen \
-  && locale-gen en_GB.utf8 \
-  && update-locale LANG=en_GB.UTF-8 \
-  && apt-get -qq update \
-  && apt-get -qq install -y --no-install-recommends locales gnupg2 \
-  && rm -rf /var/lib/apt/lists/*;
-
-
-#RUN conda install
-#
-COPY files/apt_packages /tmp/
-RUN apt-get -qq update \
+    && locale-gen en_GB.utf8 \
+    && update-locale LANG=en_GB.UTF-8 \
+    && apt-get -qq update \
     && apt-get -qq install -y \
-    $(cat /tmp/apt_packages) \
-    # Dependencies of rgl which itself is a dependency
-    && rm -rf /var/lib/apt/lists/*; \
-    # Install texlive with LaTex binaries and tools (scheme-basic)
-    wget http://mirror.ctan.org/systems/texlive/tlnet/install-tl-unx.tar.gz \
-    && mkdir /install-tl-unx; \
-    tar -xvf install-tl-unx.tar.gz -C /install-tl-unx --strip-components=1; \
-      echo "selected_scheme scheme-basic" >> /install-tl-unx/texlive.profile; \
-    /install-tl-unx/install-tl -profile /install-tl-unx/texlive.profile; \
-    rm -r /install-tl-unx; \
-    rm install-tl-unx.tar.gz; \
-#    # Install SimbaAthena ODBC drivers
-    wget -q https://s3.amazonaws.com/athena-downloads/drivers/ODBC/SimbaAthenaODBC_1.0.3/Linux/simbaathena-1.0.3.1004-1.x86_64.rpm -P /tmp \
+    locales \
+    gnupg2 \
+    alien \
+    build-essential \
+    bzip2 \
+    ca-certificates \
+    cargo \
+    curl \
+    nano \
+    vim \
+    wget \
+    git \
```

Comment:
I'd probably keep these in alphabetical order to make it easier to read/update. (not that Docker or `apt` cares ☺️ )